### PR TITLE
Dualsum

### PR DESCRIPTION
**Pull request by MihaiSurdeanu**

### CODE REVIEW

File: encoder/src/main/scala/org/clulab/scala_transformers/encoder/apps/TokenClassifierExampleApp.scala [hunk 1]
```diff
@@ -10,8 +10,8 @@ import org.clulab.scala_transformers.tokenizer.LongTokenization
 */
 
 object TokenClassifierExampleApp extends App {
-  //val tokenClassifier = TokenClassifier.fromFiles("../../scala-transformers-models/roberta-base-mtl/avg_export")
-  val tokenClassifier = TokenClassifier.fromFiles("../microsoft-deberta-v3-base-mtl/avg_export")
+  val tokenClassifier = TokenClassifier.fromFiles("/home/msurdeanu/github/scala-transformers/roberta-base-mtl/avg_export")
```
Comment (kwalcock) [inline encoder/src/main/scala/org/clulab/scala_transformers/encoder/apps/TokenClassifierExampleApp.scala]: Do you want to leave this directory in the source?

Comment (MihaiSurdeanu) [reply encoder/src/main/scala/org/clulab/scala_transformers/encoder/apps/TokenClassifierExampleApp.scala]: Not necessarily :)

**Review by kwalcock — APPROVED** [review]
It would be very difficult to check all the changes, but the important ones are the math in the encoder.  This project needs to be published in order for processors to use it.  It works best to publish from main, so it's a good time to merge this one.

**Comment by kwalcock** [comment]
I need to publish the models before it will pass.

**Pull request merged**